README.md

```diff
@@ -26,6 +26,10 @@ using [npm](https://www.npmjs.com/package/marklogic):
 ```
 npm install marklogic --save
 ```
+For Windows OS please use the below for Node Client 2.9.1: 
```

Comment:
Shouldn't this addition be coming from a separate PR?

Reply:
yes, but i don't remember why this isn't coming then :)

Reply:
Hi Marius, this is not needed. Please remove this change.